index.bs

```diff
@@ -38,36 +38,38 @@ table td, table th {
 
 # Introduction # {#introduction}
 
-The API defined in this document captures images from a photographic device referenced through a valid {{MediaStreamTrack}}. The produced image can be in the form of a {{Blob}} (see {{takePhoto()}} method) or as a {{ImageBitmap}} (see {{grabFrame()}}). Picture-specific settings can be optionally provided as arguments that can be applied to the device for the capture.
+The API defined in this document captures images from a photographic device referenced through a valid {{MediaStreamTrack}}. The produced image can be in the form of a {{Blob}} (see {{takePhoto()}} method) or as a {{ImageBitmap}} (see {{grabFrame()}}).
 
-# Image Capture API # {#imagecaptureapi"}
+Reading capabilities and settings and applying constraints is done in one of two ways depending on if it impacts the source {{MediaStreamTrack}} or not.  Picture-specific capabilities, default values and current settings can be retrieved via {{getPhotoCapabilities()}} and configured via {{takePhoto()}}'s {{PhotoSettings}} argument.  Manipulating video-related capabilities, current settings and constraints is done via the MediaStreamTrack <a href="#extensions">extension mechanism</a>.
```

Comment:
> This introduces the distinction between video settings vs. photo settings.

Correct and intended, I wanted to write about this upfront so any potential reader will notice this duality and why it's like that.

>Maybe the sentence would be even clearer if we s/source MediaStreamTrack/video MediaStreamTrack/ and s/Picture/Photo/, and maybe s/if/whether/ ?

Done all three.